src/components/SideBar.js

```diff
@@ -51,18 +52,12 @@ const defaultProfilePicture = require('../assets/default-profile-picture.jpg');
           expanded ? "w-80" : "w-[5.2rem]"
         }`}
       >
-        <div className="p-4 pb-2 h-20 bg-maroon flex justify-between items-center">
-          <img
-            loading="lazy"
-            srcSet={require("../assets/novelology_newlogo.png")}
-            className= {`bg-maroon rounded-lg overflow-hidden transition-all ${
-                expanded ? "w-16 h-14" : "w-0"
-              }`}
-          />
+        <div className="p-4 pb-2 h-20 w-full bg-maroon flex justify-between items-center">
```

Comment:
Did you only delete the logo for Sidebar?

Reply:
yes, it was in 3 places on the screen at the same time

Reply:
Oh really? That looked different on my laptop. I saw the new styling and it looks good as well.